.zed/tasks.json

```diff
@@ -0,0 +1,10 @@
+// Static tasks configuration.
```

Comment:
nit - add to gitignore?

Reply:
in other projects i've seen the tasks file be checked in, let's other devs have all of the tasks ootb, but i'm not opinionated about it